instrumentation/elasticsearch/test/opentelemetry/instrumentation/elasticsearch/instrumentation_test.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
```

Comment:
I think this file is a repetition of https://github.com/open-telemetry/opentelemetry-ruby/pull/707/files#diff-0f805288429558033dbb57131c19985abaaa93c205e141983d70eb8f9471d475

Probably makes sense to remove this in favour of the other one.

Reply:
Sure! Not sure if it was generated automatically, but happy to remove it 👍

Reply:
If you used the instrumentation generator it would have generated a file like this.   But it looks like another one was added in addition to it.  I don't have a strong preference to which one gets kept 😄